log_injection/LuckyMallari_orange-assist/or.py

```
@@ -0,0 +1,177 @@
+def index():
+    return "orange!", 200
+
+
+@app.route('/output/<f>', methods=['GET'])
+def o(f):
+    try:
+        with open(f'output/{f}') as the_file:
+            resp = the_file.read()
+            the_file.close()
+            return resp, 200
+    except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)